package.json

```diff
@@ -2,16 +2,25 @@
   "name": "restaurant-page",
   "version": "1.0.0",
   "description": "",
-  "private": true,
+  "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
     "build": "webpack"
   },
```

Comment:
Let us kindly improve our package.json file by replacing the current script with this =>
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```
so we can run common commands like "dev", "build", etc